docs/lib/restapi/fragments/js/fetch.md

```diff
@@ -26,17 +26,17 @@ API
 Example with async/await
 
 ```javascript
-async getData() { 
+function getData() { 
   const apiName = 'MyApiName';
   const path = '/path';
   const myInit = { // OPTIONAL
     headers: {}, // OPTIONAL
   };
 
-  return await API.get(apiName, path, myInit);
+  return API.get(apiName, path, myInit);
 }
 
-getData();
+const response = await getData();
```

Comment:
```suggestion
getData();
```

Reply:
I like this better:

```js
(async function() {
  await getData();
})();
```

Some developers who are not as advanced may be misled by the code example and just call `getData()`, when it really needs an `await` in front of it since it's returning a Promise. If the IIFE above is potentially confusing, then maybe just:

```js
async function main() {
  await getData();
}
```

These comments apply to the rest of your comments in this PR. I'm still of the opinion that the `head()` and `getData()` functions should return the result, per my initial suggestion.

Reply:
Im good with this as well!